### ISSUE TITLE
Supported versions of Django in Requirements section

### ISSUE THREAD

**Issue by hseritt** [nesdis/djongo#277]
It would be nice to have in the Requirements section in the README.md page which versions of Django are supported. For instance, it would be helpful to know that Django >=2.2 has some issues with crud operations in the admin. I would be happy to test things like this and contribute in a small way if helpful.

**Comment by lardissone** [comment]
I'm using version 2.1.7 and it fails on admin authentication.
Not sure if the same issue. I'm using remote databases (MongoDB Atlas, and also tried mLab).

**Comment by hseritt** [comment]
> I'm using version 2.1.7 and it fails on admin authentication.
> Not sure if the same issue. I'm using remote databases (MongoDB Atlas, and also tried mLab).

Just getting into admin with my superuser is successful, though I haven't tested other group members. Can you give the steps to reproduce? I'd like to have a look.

Thanks!

**Comment by lardissone** [comment]
I'm using Zeit's now to deploy it, but also tried the same setup in Heroku (with the Procfile in place) and same error happened.
[Here](https://now-django-example.lardissone.now.sh/_src) you can see the source, also you can try the [admin](https://now-django-example.lardissone.now.sh/admin) login with `lardissone` / `Hello1234`. And you'll see the SQL timeout error.

**Comment by hseritt** [comment]
Seems to take forever to connect. I don't think this is a problem with djongo but rather a server connection issue. Do you have another way to test if a client can connect to the backend mongo server? At this point based on what I'm seeing, I think this is simply timeout issue on either the network or mongo server side.

At any rate, I tried with Django 2.1.7 and I had no issues starting up and logging in to the admin as admin/admin.

This was the stack I used:

- dataclasses==0.6
- Django==2.1.7
- djongo==1.2.32
- pymongo==3.8.0
- pytz==2019.1
- sqlparse==0.2.4

**Comment by nesdis** [comment]
I would like to freeze the version of Django in the requirements, but people may not like it if the latest is not supported.

**Comment by hseritt** [comment]
Agreed. This is just a workaround until there's a fix. I believe there may be a fix and as soon as I can, I'll test it.
